### PR TITLE
Add hover effect to Tier Type buttons

### PR DESCRIPTION
**Pull request by jinapurapu**
https://user-images.githubusercontent.com/65002498/207742909-4c79219f-9661-496e-8662-921ef591f6ee.mov

### CODE REVIEW

File: portal-ui/src/screens/Console/Configurations/TiersConfiguration/TierTypeCard.tsx [hunk 1]
```diff
@@ -17,34 +17,40 @@
 import React from "react";
 import { Box } from "@mui/material";
 
+import { Button } from "@mui/material";
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Configurations/TiersConfiguration/TierTypeCard.tsx:20]: We are not using material-ui buttons anymore

File: portal-ui/src/screens/Console/Configurations/TiersConfiguration/TierTypeCard.tsx [hunk 2]
```diff
@@ -17,34 +17,40 @@
...
-        padding: 10,
-        background: "transparent",
+        padding: 5,
         border: "1px solid #E5E5E5",
         borderRadius: 2,
         cursor: "pointer",
-      }}
-      onClick={() => {
-        onClick(name);
+        "&:hover": {
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Configurations/TiersConfiguration/TierTypeCard.tsx:43]: In this case what you can do is to create a new class element with all the values contained in style and then assign it to each button

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by prakashsvmx — APPROVED** [review]
Tested. LGTM :+1:

**Review by oscarocastellanos — APPROVED** [review]
LGTM

**Review by bexsoft — CHANGES_REQUESTED** [review]
I'm seeing this issue on wide screens:

Screenshot 2022-12-15 at 11 41 14

**Review by bexsoft — APPROVED** [review]
Issue can be fixed in an upcoming PR

**Pull request merged**